### PR TITLE
vim-patch:2ecbe53f452e

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/2ecbe53f452e92e941aff623f6a0b72f80e43d07

### CODE REVIEW

File: runtime/ftplugin/debchangelog.vim [hunk 1]
```diff
@@ -117,7 +122,9 @@ function NewVersion()
     normal! 1G0
     call search(')')
     normal! h
-    normal! 
+    " ':normal' doens't support key annotation (<c-a>) directly.
```
Comment (smjonas) [inline runtime/ftplugin/debchangelog.vim:125]: \* doesn't

**Review by zeertzjq — COMMENTED** [review]
`matchadd()` docs wrapping doesn't match Vim

File: runtime/doc/builtin.txt [hunk 2]
```diff
@@ -6310,7 +6310,7 @@ search({pattern} [, {flags} [, {stopline} [, {timeout} [, {skip}]]]])
 <		When {stopline} is used and it is not zero this also implies
 		that the search does not wrap around the end of the file.
 		A zero value is equal to not giving the argument.
-
+				*E1285* *E1286* *E1287* *E1288* *E1289*
```
Comment (seandewar) [inline runtime/doc/builtin.txt:6313]: From 8.2.5057, which isn't ported yet

Comment (clason) [reply runtime/doc/builtin.txt:6313]: Eh, didn't mean to commit that hunk!

**Review by seandewar — APPROVED** [review]

**Pull request merged**